libraries/bezier-rs/src/utils.rs

```diff
@@ -28,6 +28,13 @@ pub enum SubpathTValue {
 	GlobalEuclideanWithinError { t: f64, error: f64 },
 }
 
+#[derive(Copy, Clone)]
+pub enum Joint {
+	Mitre,
+	Blunt,
+	Rounded,
+}
```

Comment:
Please use the names from the SVG spec: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linejoin#usage_context (note the spelling for "Miter")

Reply:
Ah good catch, I made these names up as a placeholder until I verified the real ones. I was pretty close!